build-scripts/esp-idf/wamr/Kconfig

```diff
@@ -0,0 +1,88 @@
+menu "WASM Micro Runtime"
+    choice WAMR_BUILD_TYPE
+        prompt "Build type"
+        default WAMR_BUILD_DEBUG
```

Comment:
Wondering why the default build type is debug, is it better to set it as release?